RecoJets/JetProducers/src/PileUpSubtractor.cc

```diff
@@ -378,6 +379,7 @@ void PileUpSubtractor::fillDescriptions(edm::ConfigurationDescriptions& descript
 	desc.add<double> ("Ghost_EtaMax", 5);
 	desc.add<double> ("GhostArea", 0.01);
 	desc.add<int> ("Active_Area_Repeats", 1);
+	desc.add<double> ("jetPtMin",	10.);
```

Comment:
Is this function (full PileUpSubtractor::fillDescriptions) actually doing anything?

Reply:
Surprisingly not - I left it since it was already there but I can remove it.  The RelVal and DQM run just fine without it.